src/app/components/OrderBook.tsx

```diff
@@ -139,7 +139,7 @@ export function OrderBook() {
         <div className="flex justify-end join">
           <span className="join-item mr-2">{t("grouping")} </span>
           <input
-            className="input-xs w-16 join-item"
+            className="input-xs w-16 join-item truncate"
```

Comment:
This is adding truncate to the input, ideally we'd want to give the input a min width and truncate the labels.